docusaurus/docs/deployment.md

```diff
@@ -438,9 +438,9 @@ To support `pushState`, make sure to create a `public/_redirects` file with the
 
 When you build the project, Create React App will place the `public` folder contents into the build output.
 
-## [ZEIT Now](https://zeit.co)
+## [Vercel](https://vercel.com)
 
-[ZEIT Now](https://zeit.co) is a cloud platform for websites and serverless APIs, that you can use to deploy your Create React App projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform for websites and serverless APIs, that you can use to deploy your Create React App projects to your personal domain (or a free `.now.sh` suffixed URL).

```

Comment:
`.now.sh` is also going to migrated to `.vercel.app`

WDYT?

Reply:
[Vercel timeline](https://www.notion.so/Change-Timeline-fab7622780694753a92c0a396cea7eb6) say May 20 is when `vercel.app` will be used, but it's still few weeks away. 

We could merge this and after May 20, another PR to update docs with `vercel.app` and `now` cli?